StRoot/StEvent/StETofCollection.h

```diff
@@ -59,7 +59,7 @@ class StETofCollection : public StObject {
     StSPtrVecETofHit    mETofHits;
 
 
-    ClassDef( StETofCollection, 1 )
+    ClassDef( StETofCollection, 2 )
```

Comment:
Why increase the ClassDef version when the class structure is not changed?

Reply:
In my tests, without the increment, StEtofCollection.h would load an unmodified version of StEtofHeader.h (without the additional data members in this pull request) if it was included before StEtofHeader.cxx in a Maker. The alternative would be to #include "StEtofHeader.h" explicitly in any Maker BEFORE #include "StEtofCollection.h", which also works fine. But in my opinion, this just asks for difficult-to-debug-errors down the line.